test/logout.tests.js

```diff
@@ -95,7 +95,7 @@ describe('logout route', async () => {
     assert.include(
       response.headers,
       {
-        location: `https://op.example.com/session/end?post_logout_redirect_uri=http%3A%2F%2Fexample.org&id_token_hint=${idToken}`,
```

Comment:
Logout redirect url contain one more parameter now, but I don't think it's a problem